brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-872 Improvement: Added custom font plugin support

### Description
Added custom font plugin support. fonts uploaded through custom font plugin should appear in Spectra blocks.

### Screenshots
Refer - https://share.getcloudapp.com/jkuXkmZp

### Types of changes




Improvement

### How has this been tested?
- Install Custom fonts plugin. Use this zip - https://share.getcloudapp.com/qGuOGRn7
This is the PR - https://github.com/brainstormforce/custom-fonts/pull/59/files 
- Add custom fonts from here - https://share.getcloudapp.com/llu0l6ye
- You can download woff files for font from here - https://google-webfonts-helper.herokuapp.com/fonts/open-sans?subsets=latin
- In the editor the custom fonts should load properly. Refer - https://share.getcloudapp.com/jkuXkmZp


### Checklist:
